src/features/Instructors/ManageInstructors/index.jsx

```diff
@@ -34,6 +34,8 @@ const ManageInstructors = () => {
   const previousPage = queryParams.get('previous') || 'courses';
   const isLoadingInstructors = instructorsByClass?.status === RequestStatus.LOADING;
   const isButtonDisabled = rowsSelected.length === 0;
+  const courseNameDecoded = decodeURIComponent(courseName);
+  const classNameDecoded = decodeURIComponent(className);
```

Comment:
Is really necessary create a variables for this? Why just use it directly?

Reply:
In this case is not necessary, in the other paged is because is used in more parts